interfaces/builtin/openvswitch.go

```diff
@@ -31,6 +31,7 @@ const openvswitchBaseDeclarationSlots = `
 
 const openvswitchConnectedPlugAppArmor = `
 /run/openvswitch/db.sock rw,
+/run/openvswitch/*.mgmt rw,
```

Comment:
A comment about what this mgmt files would be nice (and enumerating what some of them are). Please feel free to commit once an appropriate comment is present.

Reply:
done.

Reply:
Thanks for the added comment. Future note: once comments have been applied to the PR, we don't force push any more.